File: CommonRecycler/src/main/java/com/shuyu/bind/listener/OnBindDataChooseListener.java
package com.shuyu.bind.listener;

import java.util.List;

/**
 * 多个layoutId的情况下，判断该model使用哪个id
 * Created by guoshuyu on 2017/8/29.
 */

public interface OnBindDataChooseListener {
    int getCurrentDataLayoutId(Object object, Class classType, int position, List<Integer> ids);
}

File: CommonRecycler-Kotlin/src/main/java/com/shuyu/commonrecycler/listener/OnBindDataChooseListener.kt
package com.shuyu.commonrecycler.listener

/**
 * 多个layoutId的情况下，判断该model使用哪个id
 * Created by guoshuyu on 2017/8/29.
 */

open interface OnBindDataChooseListener {
    fun getCurrentDataLayoutId(`object`: Any, classType: Class<*>, position: Int, ids: List<Int>): Int
}
